luigi_pipeline/lib/hail_tasks.py

```diff
@@ -75,8 +75,16 @@ def run(self):
 
     def import_vcf(self):
         # Import the VCFs from inputs. Set min partitions so that local pipeline execution takes advantage of all CPUs.
+        recode = {}
+        if self.genome_version == "38":
+            recode = {f"{i}": f"chr{i}" for i in (list(range(1, 23)) + ['X', 'Y'])}
```

Comment:
Wouldn't we want both build versions to map to the same set of chromosome names?

Reply:
That's what you'd think, but the official names for GRCh37 contigs are 1,2,3,4 and GRCh38 is chr1, chr2, chr..

Reply:
🤦